src/modules/update.js

```diff
@@ -0,0 +1,12 @@
+export default function updateList(array, id) {
+  array.map((item) => {
+    if (item.id === id) {
+      if (item.completed === false) {
+        item.completed = true;
+      } else {
+        item.completed = false;
+      }
+    }
+    return item;
+  });
+}
```

Comment:
- [ ] Great job on implementing the completed functionality for a task @olanikegloria, The only issue I can see is when I reload the page the task that are completed are no more marked as completed. This prevents the user from identifying which tasks are completed and which are not. Please be reminded I have noticed the change is stored in the local storage but the UI is not getting updated. 


https://github.com/olanikegloria/to-do-list/assets/69077061/21faf2f9-730b-4bef-86c2-7ac8d14726ed